staging/src/k8s.io/apimachinery/pkg/api/equality/semantic.go

```diff
@@ -22,6 +22,7 @@ import (
 	"k8s.io/apimachinery/pkg/conversion"
 	"k8s.io/apimachinery/pkg/fields"
 	"k8s.io/apimachinery/pkg/labels"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
This will add a dependency to k8s.io/api from k8s.io/apimachinery, which would be cyclical, since k8s.io/api already depends on k8s.io/apimachinery

Reply:
If you need an equality semantic which understands things in k8s.io/api/core/v1, it either has to live in core/v1, live somewhere else, or you have to move those parts of k8s.io/api/core/v1 to k8s.io/apimachinery